compiler/testData/diagnostics/tests/resolve/invoke/errors/unsafeCallWithInvoke.fir.kt

```diff
@@ -4,7 +4,7 @@
 operator fun String.invoke(i: Int) {}
 
 fun foo(s: String?) {
-    <!INAPPLICABLE_CANDIDATE!>s<!>(1)
+    <!UNSAFE_CALL!>s<!>(1)
 
-    <!INAPPLICABLE_CANDIDATE!>(s ?: null)(1)<!>
+    <!UNSAFE_CALL!>(s ?: null)(1)<!>
```

Comment:
OI/NI behave differently: https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/resolve/invoke/errors/unsafeCallWithInvoke.kt#L9

Assuming NI is default in FIR, I think we should report `UNSAFE_IMPLICIT_INVOKE_CALL` here?